code/modules/mob/living/simple_animal/hostile/megafauna/stalwart.dm

```diff
@@ -0,0 +1,176 @@
+/mob/living/simple_animal/hostile/megafauna/stalwart
+	name = "stalwart"
+	desc = "A graceful, floating automaton. It emits a soft hum."
+	health = "3000"
+	maxHealth = "3000"
+	attacktext = "zaps"
+	attack_sound = 'sound/effects/empulse.ogg'
+	icon_state = "stalwart"
+	icon_living = "stalwart"
+	icon_dead = ""
+	friendly = "scans"
+	icon = 'icons/mob/lavaland/64x64megafauna.dmi'
+	speak_emote = list("screeches")
+	armour_penetration = 40
+	melee_damage_lower = 40
+	melee_damage_upper = 40
+	speed = 5
+	move_to_delay = 5
+	ranged = TRUE
+	del_on_death = TRUE
+	pixel_x = -16
+	internal_type = /obj/item/gps/internal/stalwart
+	loot = list(/obj/structure/closet/crate/sphere/stalwart)
+	deathmessage = "erupts into blue flame, and screeches before violently shattering."
+	deathsound = 'sound/voice/borg_deathsound.ogg'
+	internal_type = /obj/item/gps/internal/stalwart
+	var/charging = FALSE
+	var/revving_charge = FALSE
+
+/mob/living/simple_animal/hostile/megafauna/stalwart/OpenFire()
+	ranged_cooldown = world.time + 120
```

Comment:
this is also in deciseconds, means 12 seconds between each attack which might be a bit long